lib/std/testing.zig

```diff
@@ -400,6 +400,56 @@ pub fn tmpDir(opts: std.fs.Dir.OpenDirOptions) TmpDir {
     };
 }
 
+/// caller owns memory
+pub fn tmpDirPath(alloc: std.mem.Allocator, tmpdir: *TmpDir) ![]u8 {
```

Comment:
Any reason why this isn't in the `TmpDir` struct as something like:

```zig
pub fn getFullPath(self: TmpDir, alloc: std.mem.Allocator) ![]u8
```

Reply:
fixed.